pkg/webui/console/api/index.js

```diff
@@ -89,9 +89,8 @@ export default {
     },
   },
   devices: {
-    list: stubs.devices.list,
-    search: stubs.devices.search,
-    get: stubs.device.get,
+    list: ttnClient.Applications.Devices.getAll.bind(ttnClient.Applications.Devices),
```

Comment:
gives `Uncaught TypeError`

Reply:
Please try building the sdk again and then do
`make js.build-dll -B; make js.serve`

Reply:
issue persists